dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -764,6 +764,10 @@ error.checkpoint.failed.selinux.netdiag=CWWKE0963E: The server checkpoint reques
 error.checkpoint.failed.selinux.netdiag.explanation=A checkpoint request for the server failed because netlink system calls were unsuccessful.
 error.checkpoint.failed.selinux.netdiag.useraction=If SELinux is enabled in enforcing mode, netlink system calls might be blocked by the SELinux "virt_sandbox_use_netlink" policy setting. Either disable SELinux or enable the netlink system calls with the "setsebool virt_sandbox_use_netlink 1" command.
 
+error.checkpoint.restore.no.recovery=CWWKE0964E: Restoring the checkpoint server process failed. Check the {0} log to determine why the checkpoint process was not restored. The server did not launch.
+error.checkpoint.restore.no.recovery.explanation=An error occurred restoring a checkpoint server process. Checkpoint restore failure recovery is disabled so the server did not launch.
+error.checkpoint.restore.no.recovery.useraction=Check the logs to determine why the checkpoint process was not restored. To enable restore failure recovery, unset the environment variable CRIU_RESTORE_DISABLE_RECOVERY.
```

Comment:
Shouldn't this have the same sentence as:
```
error.checkpoint.failed.useraction=To determine why the checkpoint failed, inspect the log files in the checkpoint directory that is located in the server logs directory.
```

This elevates the property to an external.  I should bring this up in the UFO.  I am thinking we may want to flip the default of this to `true`.  In that case the `useraction` would be:

```
To enable restore failure recovery set the environment variable CRIU_RESTORE_DISABLE_RECOVERY to "true".
```

Reply:
For useraction I followed the pattern of the exisitng error message:
info.checkpoint.restore.failed.useraction=Check the logs to determine why the checkpoint process was not restored.